docs/conf.py

```diff
@@ -282,7 +294,7 @@ def _skip(self, word):
     # Latex figure (float) alignment
     #
     # 'figure_align': 'htbp',
-}
+}  # type: dict[str, str]
```

Comment:
🤔 I don't think this type hint should work

```suggestion
}  # type: Dict[str, str]
```

Reply:
👀 awesome!